numpy/lib/npyio.py

```diff
@@ -2225,6 +2225,8 @@ def ndfromtxt(fname, **kwargs):
     """
     Load ASCII data stored in a file and return it as a single array.
 
+    .. deprecated:: np.ndfromtxt is deprecated
+
```

Comment:
needs a version number, see the other deprecation notices

Reply:
I was wondering if it should be 1.17?